{"repository":"commons-math","prod_path":"src/main/java/org/apache/commons/math/exception/util/ExceptionContext.java","test_path":"src/test/java/org/apache/commons/math/exception/util/ExceptionContextTest.java","prod_time":"2011-09-04 22:38:40","test_time":"2011-09-04 22:38:40","add_annotation_line":0,"add_call_line":2,"add_classname_line":0,"add_condition_line":0,"add_field_line":4,"add_import_line":0,"add_packageid_line":0,"add_parameter_line":1,"add_return_line":1,"del_annotation_line":0,"del_call_line":0,"del_classname_line":0,"del_condition_line":0,"del_field_line":3,"del_import_line":0,"del_packageid_line":0,"del_parameter_line":0,"del_return_line":0,"label":"POSITIVE","pro_Commit":"d696293435c04a8948c873fee4ed3a23536d7a26","test_Commit":"d696293435c04a8948c873fee4ed3a23536d7a26","true_label":null}